packages/har-remix/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "har-remix",
+  "version": "2.0.0-beta.2",
+  "description": "Server HAR archive with loose matching and hooks to alter responses.",
+  "keywords": [
+    "benchmarking",
+    "har",
+    "server"
+  ],
+  "repository": "github:devtrace/har-remix",
+  "license": "BSD-2-Clause",
+  "author": "Kris Selden <kris.selden@gmail.com>",
+  "files": [
+    "dist",
+    "!dist/test",
+    "src",
+    "!src/test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "generate-har": "node scripts/generate_har_interfaces.js",
+    "prepare": "tsc",
+    "test": "echo \"TODO add tests for har-remix\""
+  },
+  "dependencies": {
+    "mime-types": "^2.1.20"
+  },
+  "devDependencies": {
+    "@krisselden/har-schema": "^0.1.0",
```

Comment:
should we also move these into the tracerbench org?

Reply:
sure, it is a fork of an npm module since the owner was unresponsive to my PR